cypress.config.cjs

```diff
@@ -0,0 +1,18 @@
+const { defineConfig } = require('cypress')
```

Comment:
We are using ESM with ES6, you need to change the require by import

Reply:
do you know where I need to import require from?

Reply:
figured it out

Reply:
@OOCAZ You must change also the _export_.
It's better to change extension in `.mjs` for ESM ([souce](https://docs.cypress.io/guides/references/configuration#Configuration-File)).